test/runtime/Policies.go

```diff
@@ -874,6 +874,54 @@ var _ = Describe("RuntimeValidatedPolicies", func() {
 
 	})
 
+	It("Enforces ToFQDNs policy", func() {
+		By("Importing policy with ToFQDN rules")
+		// notaname.cilium.io never returns IPs, and is there to test that the
+		// other name does get populated.
+		fqdnPolicy := `
+[
+  {
+    "endpointSelector": {
+      "matchLabels": {
+        "container:id.app1": ""
+      }
+    },
+    "egress": [
+      {
+				"toPorts": [{
+          	"ports":[{"port": "53", "protocol": "ANY"}]
+        }]
+      },
+      {
+        "toFQDNs": [
+          {
+            "matchName": "cilium.io"
+          },
+          {
+            "matchName": "notaname.cilium.io"
+          }
+        ]
+      }
+    ]
+  }
+]`
+		_, err := vm.PolicyRenderAndImport(fqdnPolicy)
+		Expect(err).To(BeNil(), "Unable to import policy: %s", err)
+
+		By("Testing that ToFQDN enables policy enforcement")
+
+		By("Denying egress to IPs of non-ToFQDN DNS names that are looked up")
+		for _, blockedTarget := range []string{"www.cilium.io", "google.com", "yahoo.com", "1.1.1.1", "127.0.0.1"} {
+			res := vm.ContainerExec(helpers.App1, helpers.CurlFail(blockedTarget))
+			res.ExpectFail("Curl succeeded against blocked DNS name " + blockedTarget)
+		}
+
+		By("Allowing egress to IPs of specified ToFQDN DNS names")
+		allowedTarget := "cilium.io"
+		res := vm.ContainerExec(helpers.App1, helpers.CurlWithHTTPCode(allowedTarget))
+		res.ExpectContains("301", fmt.Sprintf("Cannot access "+allowedTarget+"%v", res.OutputPrettyPrint()))
```

Comment:
you don't need to use fmt.Sprintf, I would change this line to: 
```
res.ExpectContains("301", "Cannot access to %q err: %s", allowedTarget, res.OutputPrettyPrint())
```

Reply:
Oh, excellent. I'll change it! thanks